components/eamxx/src/control/atmosphere_surface_coupling_exporter.cpp

```diff
@@ -207,47 +249,94 @@ void SurfaceCouplingExporter::run_impl (const double dt)
 // =========================================================================================
 void SurfaceCouplingExporter::do_export(const double dt, const bool called_during_initialization)
 {
-  using policy_type = KT::RangePolicy;
+  if (m_num_const_exports>0) {
+    do_export_constant(dt,called_during_initialization);
+  }
+  if (m_num_eamxx_exports>0) {
+    do_export_from_eamxx(dt,called_during_initialization);
+  }
+
+  // Finish up exporting vars
+  do_export_to_cpl(called_during_initialization);
+}
+// =========================================================================================
+void SurfaceCouplingExporter::do_export_constant(const double dt, const bool called_during_initialization)
+{
+  // Cycle through those fields that will be set to a constant value:
+  for (int i=0; i<m_num_scream_exports; ++i) {
+    if (m_export_source(i)==CONSTANT) {
+      std::string fname = m_export_field_names[i];
+      const auto field_view = m_helper_fields.at(fname).get_view<Real*>();
+      Kokkos::deep_copy(field_view,m_export_constants(i));
```

Comment:
Another one here. It looks like `m_export_constants` does not need to be a dev view, btw. You could just use a std::vector, or a std::map.